lms/djangoapps/teams/static/teams/js/views/team_utils.js

```diff
@@ -1,70 +1,70 @@
         return {
 
             /**
              * Convert a 2d array to an object equivalent with an additional blank element
              *
-             * @param options {Array.<Array.<string>>} Two dimensional options array
+             * @param {Array.<Array.<string>>} options Two dimensional options array
              * @returns {Object} Hash version of the input array
              * @example selectorOptionsArrayToHashWithBlank([["a", "alpha"],["b","beta"]])
              * // returns {"a":"alpha", "b":"beta", "":""}
              */
-            selectorOptionsArrayToHashWithBlank: function (options) {
+            selectorOptionsArrayToHashWithBlank: function(options) {
                 var map = _.object(options);
-                map[""] = "";
+                map[''] = '';
                 return map;
             },
 
-            teamCapacityText: function (memberCount, maxMemberCount) {
-                return interpolate(
+            teamCapacityText: function(memberCount, maxMemberCount) {
+                return StringUtils.interpolate(
                     // Translators: The following message displays the number of members on a team.
                     ngettext(
-                        '%(memberCount)s / %(maxMemberCount)s Member',
-                        '%(memberCount)s / %(maxMemberCount)s Members',
+                        '{memberCount} / {maxMemberCount} Member',
+                        '{memberCount} / {maxMemberCount} Members',
                         maxMemberCount
                     ),
-                    {memberCount: memberCount, maxMemberCount: maxMemberCount}, true
+                    {memberCount: memberCount, maxMemberCount: maxMemberCount}
                 );
             },
 
             isUserMemberOfTeam: function(memberships, requestUsername) {
                 return _.isObject(
-                    _.find(memberships, function(membership)
-                    {
+                    _.find(memberships, function(membership) {
                         return membership.user.username === requestUsername;
                     })
                 );
             },
 
-            hideMessage: function () {
+            hideMessage: function() {
                 $('#teams-message').addClass('.is-hidden');
             },
 
-            showMessage: function (message, type) {
-                var messageElement = $('#teams-message');
+            showMessage: function(message, type) {
+                var $message = $('#teams-message');
                 if (_.isUndefined(type)) {
                     type = 'warning';
                 }
-                messageElement.removeClass('is-hidden').addClass(type);
-                $('.teams-content .msg-content .copy').text(message);
-                messageElement.focus();
+                $message.removeClass('is-hidden').addClass(type);
+                HtmlUtils.setHtml($('.teams-content .msg-content .copy'), message);
```

Comment:
Note that this usage of ``$()`` isn't creating an element, but is finding a matching one. It is amazing all of the different behaviors that ``$`` supports based upon the type of arguments it receives.

The only usages that we need to switch out are when ``$()`` is used to create an element, e.g.

    $('<div></div>')

The idea is that ``HtmlUtils.element`` is safer because it only works with HTML snippets, i.e.

    HtmlUtils.element(HtmlUtils.HTML('<div></div>'));